index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section id="welcome-section">
+        <h1>Hey I am Mimic</h1>
+    </section>
+    <section id="projects">
+        <p class="project-tile">Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque eum, perferendis odio aliquid dolorem dolores officiis, ipsum rerum totam provident aut, nihil ut velit repellat quibusdam incidunt corrupti rem consequuntur.</p>
+        <a href="#">Tribute Page</a>
```

Comment:
- [ ] Please create a <footer and add your "Tribute Page" <a element there. It is good practice to group these sort of links in a footer element to improve the structure of your code.